pyproject.toml

```diff
@@ -75,13 +76,14 @@ Flask-Cors = {version = "^3.0.10", optional = true}
 uWSGI = {version = "^2.0.21", optional = true}
 Jinja2 = {version = ">2.11.3", optional = true}
 asyncio = "^3.4.3"
-jaeger-client = {version = "^4.8.0", optional = true}
-flask-opentracing = {version="^2.0.0", optional = true}
```

Comment:
flask/jaeger are deprecated. Swapping over to the otel-api/sdk libraries instead.